tests/js/open-behavior-tests.js

```diff
@@ -156,7 +156,7 @@ module.exports = {
                 otherUserRealm.write(() => {
                     otherUserRealm.create(schemas.TestObject.name, { doubleCol: 42.133 });
                 });
-                return otherUserRealm.syncSession.uploadAllLocalChanges().then(() => {
+                return otherUserRealm.App.SyncSession.uploadAllLocalChanges().then(() => {
```

Comment:
Isn't `uploadAllLocalChanges` an instance function, called on a class here?
(If so, this is done in more places)

Reply:
It is a bad search/replace - good catch!